### DISCUSSION TITLE
Blade component context

### DISCUSSION THREAD

**Discussion by dsazup** [laravel/framework#33128] [General]
Hey. I remember in blade-x package by spatie there was `context` component which would allow to pass some data to other components. So for example something like this would be possible.

```blade
<x-form>
 <context :model="$user">
    <x-input name="email"/>
    <x-input name="name"/>
  </context>
</x-form>
```

as you can see we don't need to pass $user to each component. Wondering if something like this is possible. Looking at the code I cannot see any functionality like that, but maybe there is an easy way to do it?

**Comment by driesvints** [comment]
There's no context feature in Blade components. You'll need to explicitly pass the data to the specific components:

```
<x-form>
    <x-input name="email" :model="$user"/>
    <x-input name="name" :model="$user"/>
</x-form>
```
